solutions/standard/variables.tf

```diff
@@ -315,3 +315,25 @@ variable "admin_pass_sm_secret_name" {
   description = "The name of a new elasticsearch administrator secret. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
   default     = "elasticsearch-admin-password"
 }
+
+##############################################################
+# Kibana Configuration
+##############################################################
+
+variable "code_engine_project_name" {
+  description = "Name of the code engine project to deploy Kibana. If no value is passed, a new code engine project will be created."
+  type        = string
+  default     = null
+}
+
+variable "enable_kibana_dashboard" {
+  type        = bool
+  description = "Set it true to deploy Kibana in code engine. NOTE: Kibana image is coming direcly from the official registry (https://www.docker.elastic.co/) and not certified by the IBM."
+  default     = false
+}
+
+variable "es_full_version" {
+  description = "(Optional) Full version of the Elasticsearch instance in the format `x.x.x` to deploy Kibana dashboard. If no value is passed, data lookup will fetch the full version using the Elasticsearch API.  "
```

Comment:
Its not very clear here what `es_full_version` means. As far as I can see, its used to determine the version of kibana that is deployed. This should be made more clear, as its only applicable when deploying kibana

Reply:
`es_full_version` is the version of the  elasticsearch, which is required to deploy Kibana as this helps in pulling Kibana Docker image 
`docker pull docker.elastic.co/elasticsearch/elasticsearch:<es_full_version>`

[Ideally, you should be running Elasticsearch and Kibana with matching version numbers. If your Elasticsearch has an older version number or a newer major number than Kibana, then Kibana will fail to run.](https://github.com/elastic/kibana?tab=readme-ov-file#version-compatibility-with-elasticsearch)

Reply:
Its not me you need to explain it to, its the variable description :)

Was there a specific ask to expose this? If the recommendation is to use same version as Elasticsearch, why would we not just always do that?

Reply:
So there is https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/0df97ee41fca1b08b9d8c077f75a403f51de944a/solutions/standard/variables.tf#L45 which has only `major.minor` version to create ES instance but it is required to have `major.minor.patch` full version same AS ES instance for the docker to pull Kibana image. I have update the varible description with the reference URL.